components/LeagueCard/LeagueCard.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { cn } from '@/utils/utils';
+import Image from 'next/image';
+import placeholderImage from './placeholderImage.svg';
+
+interface ILeagueCardProps extends React.HTMLAttributes<HTMLDivElement> {
+  isEliminated?: boolean;
+}
+
+const LeagueCard = React.forwardRef<HTMLDivElement, ILeagueCardProps>(
+  ({ className, isEliminated = false, ...props }, ref) => (
+    <div
+      data-testid="LeagueCard"
+      ref={ref}
```

Comment:
What is the benefit of using ref?

Reply:
1. This is default behavior for shadcn/ui components
2. Based on my research, using `forwardRef` and `ref` allows UI libraries to provide the end user more flexibility.  It allows us to control DOM elements directly.